examples/fcma/mvpa_voxel_selection.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from brainiak.fcma.mvpa_voxelselector import MVPAVoxelSelector
+from sklearn import svm
+import sys
+import os
+from mpi4py import MPI
+import logging
+from file_io import generate_epochs_info
+from file_io import write_nifti_file
+import nibabel as nib
+import numpy as np
+
+format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+# if want to output log to a file instead of outputting log to the console,
+# replace "stream=sys.stdout" with "filename='fcma.log'"
+logging.basicConfig(level=logging.INFO, format=format, stream=sys.stdout)
+logger = logging.getLogger(__name__)
+
+"""
+example running command:
+mpirun -np 2 python mvpa_voxel_selection.py /Users/yidawang/data/face_scene/raw nii.gz /Users/yidawang/data/face_scene/mask.nii.gz
+                        data/fs_epoch_labels.npy 18
+"""
```

Comment:
How about providing some sample data like in SRM?
https://github.com/IntelPNI/brainiak/blob/master/examples/funcalign/download-data.sh

If the files are small enough (a few kB), you can even add them to Git.